app/navigation/Navigator.scala

```diff
@@ -37,7 +41,14 @@ class Navigator @Inject()() {
     case None        => routes.SessionExpiredController.onPageLoad()
   }
 
+  private def artSoldOverThresholdRouteEdit(answers: UserAnswers): Call = answers.get(BoughtOrSoldOverThresholdPage) match {
```

Comment:
Do we want to use "Edit", to keep inline with the current service, or "Check", to be consistent with the new scaffolding?

Reply:
All looks good to me. Spun it up & poked about a bit too. We may hit further hurdles with more complicated mini-flows but this one works just fine.

Reply:
@owen-opencast - will change to "Check". Let's be uniform in our approach.